client/state/selectors/README.md

```diff
@@ -9,5 +9,49 @@ When adding a new selector to this directory, make note of the following details
 
 - Each new selector exists in its own file, named with [kebab case](https://en.wikipedia.org/wiki/Kebab_case) (dash-delimited lowercase)
 - There should be no more than a single default exported function per selector file
+- The JSDoc header for the default exported function should contain an alias tag with the name of the function in it and that should match the camelCase conversion of the filename itself
```

Comment:
It was largely to prevent writing some error-probe code to identify situations like you mentioned where the default export is the name of a variable (even a function variable) or is composed/wrapped by something. Ultimately the connection between docblock and Code was severed unless _some_ tag was explicitly given.

I'm all for giving this up if we can reliably and accurately do it. I spent way more time than I wanted to though trying to make that happen and failed.